src/libcore/fmt/mod.rs

```diff
@@ -272,10 +272,12 @@ impl<'a> Arguments<'a> {
 /// safely be done so, so no constructors are given and the fields are private
 /// to prevent modification.
 ///
-/// The `format_args!` macro will safely create an instance of this structure
+/// The [`format_args!`] macro will safely create an instance of this structure
 /// and pass it to a function or closure, passed as the first argument. The
 /// macro validates the format string at compile-time so usage of the `write`
 /// and `format` functions can be safely performed.
+///
+/// [`format_args!`]: ../../std/macro.format_args!.html
```

Comment:
Should we link to `../../std/macro.format_args!.html` or `../../std/macro.format_args.html`? (note the lack of the `!` in the second version).

The latter seems to not work, but I might be doing something wrong....